src/utils/simple_optim.jl

```diff
@@ -15,6 +15,10 @@ end
 struct GoldenSectionOpt <: UnivariateOpt end
 struct BrentsMethodOpt <: UnivariateOpt end
 
+function univariate_modifier(fun, modifier::UnivariateOpt; a=0, b=1, transform=unit_transform, tol=10e-5, kwargs...)
```

Comment:
unit_transform was default previously. I think if we use unit_transform (with a=0 and b=1) it can search the entire positive x axis while identity only searches between a and b, and since we are not certain of the branch lengths I think it is better to keep unit_transform. What do you think?

Reply:
You make a good point here. If `a=0` and `b=1` is default, it makes sense to have `transform=unit_transform` as default.